android/src/app/organicmaps/editor/EditorFragment.java

```diff
@@ -410,6 +410,7 @@ private void initViews(View view)
     categoryBlock.setOnClickListener(this);
     // TODO show icon and fill it when core will implement that
     UiUtils.hide(categoryBlock.findViewById(R.id.icon));
+    UiUtils.hide(categoryBlock.findViewById(R.id.icon_up));
```

Comment:
There is no icon_up view in the editor fragment, so this line causes the app to crash when you press edit on a POI page. Is this line needed for something? It works fine with it commented out :)